### PR TITLE
Fix windows tip appearing in code block

### LINKED ISSUE #12074: Markup incorrectly surrounding tip with codeblock tags
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

A windows-specific tip for [Using Git/Splitting a subfolder](https://docs.github.com/en/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository) is being formatted as a code block (possibly due to the indentation in the markup).

### What changes are you suggesting?

Correcting the extra indentation in the markup should fix this. The html code is correctly being generated, but are surrounded by html tags for a code block. As it lies inside a nested list item, the spaces before the tip tag are most likely causing it to display as a code block.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Ste1io**
### Why:

Closes #12074 

### What's being changed:

Removed unnecessary indentation for the tip.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md [hunk 1]
```diff
@@ -27,24 +27,24 @@ If you create a new clone of the repository, you won't lose any of your Git hist
   $ cd <em>REPOSITORY-NAME</em>
   ```
 5. To filter out the subfolder from the rest of the files in the repository, run [`git filter-repo`](https://github.com/newren/git-filter-repo), supplying this information:
-    - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
+   - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
```
Comment (inderashah007) [inline content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md:30]:
```suggestion
   - `FOLDER-NAME`: The folder within your project where you'd like to create a separate repository.
```

Comment (StelioKontosXBL) [reply content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md:30]: I used 3 spaces per [GitHub Flavored Markdown Spec 5.2 ex 274](https://github.github.com/gfm/#example-274).

Comment (StelioKontosXBL) [reply content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md:30]: My apologies, just looked again and realized your revision is correct; had it in my head I had already set it to 3 spaces, but I guess I missed that line. Thanks.

**Comment by ramyaparimi** [comment]
@StelioKontosXBL 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by mchammer01 — APPROVED** [review]
@StelioKontosXBL - thanks for fixing the issue with the Windows tip ✨

**Pull request merged**